### PR TITLE
methods updated

### PR DESCRIPTION
**Pull request by jnfaerch**

### CODE REVIEW

**Comment by jnfaerch** [comment]
Line 30 will be updated

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,23 +1,43 @@
...
+    end
   end
 
   def edit
+    @booking = Booking.find(booking_params)
   end
 
   def update
+    @booking = Booking.find(booking_params)
+    @task.update(booking_params)
```
Comment (juandigt) [inline app/controllers/bookings_controller.rb]: task??

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -1,23 +1,43 @@
...
+    redirect_to bookings_path
   end
 
   def destroy
   end
+
+  private
+
+  def booking_params
+    params.require(:booking)
```
Comment (juandigt) [inline app/controllers/bookings_controller.rb]: should we permit user_id or space_id?

**Review by juandigt — APPROVED** [review]
looks good!

**Pull request merged**